Document ipc configuration file (#47)
ConsenSys/doc.goquorum: docs/Concepts/Privacy/PrivateTransactionManager.md

The Private Transaction Manager is a separate component that is concerned with the storing and distribution
of encrypted private transaction data between recipients of a private transaction.

The Private Transaction Manager has two distinct components:

* Transaction Manager
* Enclave

To enable private transactions, use the `PRIVATE_CONFIG` environment variable when starting a GoQuorum

> node to provide the path to the Privacy Manager's `.ipc` socket. There are two ways to do this, which are described below.

Reviewer: ```suggestion
node to provide the path to the Privacy Manager's `.ipc` socket.

Use the `PRIVATE_CONFIG` environment variable either with a direct path to the IPC socket or with a path to a configuration file as explained in the following sections. 
```

## Direct IPC connection configuration

> This is the normal method for specifying the IPC socket:

Reviewer: ```suggestion
This is the basic method for specifying the IPC socket:
```

```bash
export PRIVATE_CONFIG=path/to/tm.ipc
```

## Using an IPC connection configuration file

> Alternatively, a TOML configuration file can be used to specify the details of the IPC socket:

Reviewer: ```suggestion
Use a TOML configuration file can to specify more options of the IPC socket:
```

```bash
export PRIVATE_CONFIG=path/to/ipc-config-file.toml
```

> Using this method allows the configuration of timeouts for the ipc socket. Here is an example of the TOML file:

Reviewer: ```suggestion
Configuration file provides the ability to configure timeouts for the ipc socket.
```

The Private Transaction Manager is a separate component that is concerned with the storing and distribution
of encrypted private transaction data between recipients of a private transaction.

The Private Transaction Manager has two distinct components:

* Transaction Manager
* Enclave

To enable private transactions, use the `PRIVATE_CONFIG` environment variable when starting a GoQuorum
node to provide the path to the Privacy Manager's `.ipc` socket.

> Use the `PRIVATE_CONFIG` environment variable either with a direct path to the IPC socket or with a path to a configuration file as explained in the following sections. 

Author: @NicolasMassart Your amendment is now duplicating the text "Use the `PRIVATE_CONFIG` environment variable'".

Reviewer: We can repeat this kind of things if it makes the text easier to read. Sometimes using references like "this" or "previous" is not clear enough for non english speakers of when translated using Google Translate (lof of Chinese users using Translate for instance).

## Direct IPC connection configuration

> This is the basic method for specifying the IPC socket:

Author: I would have preferred to leave this saying that this is the "normal" method as we want to discourage folks from using the more complex method of having a TOML file unless it's really necessary.

Reviewer: Why? Basic means you fulfil basic needs which is the usual use case, but normal implies that the other one is not normal and thus a bit hacky.

```bash
export PRIVATE_CONFIG=path/to/tm.ipc
```

## Using an IPC connection configuration file

> Use a TOML configuration file can to specify more options of the IPC socket:

Author: Mistake in the grammar - I think it should read:
"Use a TOML configuration file to specify more options for the IPC socket:"

Reviewer: Yes will fix that, I'm making changes right now, ignore for the next 20 mins ;)
